### DISCUSSION TITLE
It looks bad.

### DISCUSSION THREAD

**Discussion by luckfunc** [community/community#63101] [Mobile] [answered]
1. Enter the GitHub app.
2. Click on "Profile."
3. Click on "Following."
4. Click on a person you're following. 
5. Access their homepage.

![image](https://github.com/community/community/assets/85545304/cd2f9091-ccb9-42ff-8b25-afcf8bacccb5)![image](https://github.com/community/community/assets/85545304/76322d23-090d-413e-8094-8e7cb62ef00e)![image](https://github.com/community/community/assets/85545304/351eadc3-e928-460e-bd2b-6f2ebd620d5f)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.124.0 (105848740)
Device: iPhone 11 (iOS 15.3)
</details>

**Comment by mattruggio** [comment, marked as answer]
Hey @Ddkkkk11, thank you very much for these wonderful reproduction steps! This is a bug specifically related to running the GitHub application on iOS 15 only (iOS 16 looks to function correctly). This specific iOS 15 bug has been acknowledged by our mobile team and will be looking to put out a fix in a future release.

**Reply by chips47310** [reply]
#
